packages/inab-web/src/styles/tables.scss

```diff
@@ -1,6 +0,0 @@
-.table > tbody > tr > td.right,
```

Comment:
Did you replace the `td.right` and `th.tright`? :)

Reply:
the .right class wasn't used anywhere lol

Reply:
Oh wow nevermind it's actually used !